spring-integration-mail/src/test/java/org/springframework/integration/mail/ImapMailReceiverTests.java

```diff
@@ -193,6 +237,11 @@ public void testIdleWithServerGuts(ImapMailReceiver receiver, boolean mapped, bo
 		adapter.setTaskScheduler(taskScheduler);
 		adapter.setReconnectDelay(1);
 		adapter.start();
+		MimeMessage message = GreenMailUtil.createTextEmail("Foo <foo@bar>", "Bar <bar@baz>", "Test Email", "foo\r\n", imapIdleServer.getImap().getServerSetup());
```

Comment:
For better readability we recommend to have code lines not longer than `120` symbols.